src/uct/ib/ud/base/ud_iface.c

```diff
@@ -350,13 +260,67 @@ static void uct_ud_iface_timer(int timer_id, int events, void *arg)
     uct_ud_iface_async_progress(iface);
 }
 
+static const void *
+uct_ud_iface_conn_match_get_peer_address(const ucs_conn_match_elem_t *elem)
+{
+    uct_ud_ep_t *ep = ucs_container_of(elem, uct_ud_ep_t, conn_match);
+    return uct_ud_ep_get_peer_address(ep);
+}
```

Comment:
can we unite uct_ud_iface_conn_match_get_peer_address and uct_ud_ep_get_peer_address to one function?